packages/cli/src/commands/element.ts

```diff
@@ -370,13 +399,21 @@ const cloneDef = createWorkspaceCommand({
       },
     ],
     keyedOptions: [
+      ENVIRONMENT_OPTION,
       {
         name: 'toEnvs',
         description: 'The environment(s) to clone to',
         type: 'stringsList',
-        required: true,
+        required: false,
+      },
+      {
+        name: 'toAllEnvs',
+        required: false,
+        description: 'Clone to all environments but for the source environment',
```

Comment:
nit: I think we can just say
```suggestion
        description: 'Clone to all other environments',
```
(not even sure the `other` is needed here), the element will exist in the source env at the end of this operation either way